src/main/java/uk/gov/hmcts/reform/hmc/api/model/request/PanelRequirements.java

```diff
@@ -0,0 +1,17 @@
+package uk.gov.hmcts.reform.hmc.api.model.request;
+
+
+public class PanelRequirements {
+
+    //private List<String> roleType;
```

Comment:
is this class needed?

Reply:
Hi @yogendraU its a mandatory class panel requirements is needed as input for hmc-cft client, currently am removing comments and using this fields for temporary fix to showcase in demo